frontend/src/pages/Login.jsx

```diff
@@ -42,18 +42,32 @@ const Login = () => {
     toast[type](message, { position: toast.POSITION.TOP_CENTER });
   };
 
-  useEffect(() => {
-    // Close toast when isLoggedIn changes
-    if (authCtx.isLoggedIn) {
-      const closeToast = () => toast.dismiss();
-      setTimeout(closeToast, 3000); // Adjust the timeout as needed
+  const demoLoginHandler = async () => {
+    setUsername('bhavy@gmail.com');
+    setPassword('123');
```

Comment:
this state updation is unnecessary

Reply:
okay removed it

Reply:
can you say why it is not?
context ma store that chhe ?